apps/raw2vdc/raw2vdc.cpp

```diff
@@ -101,7 +102,7 @@ int read_data(FILE * fp,
     if (type.compare("float32") == 0) {
         float *sptr = (float *)buffer;
         for (size_t i = 0; i < n; i++) { *dptr++ = (float)*sptr++; }
-    } else if (type.compare("float64") == 0) {
+    } else if ((type.compare("float64") == 0) || (type.compare("double") == 0)) {
         double *sptr = (double *)buffer;
         for (size_t i = 0; i < n; i++) { *dptr++ = (float)*sptr++; }
```

Comment:
Just a sanity check, does this code mean that if the input data is in double precision, it's truncated to single precision when read in?

Reply:
No, it just treats the "double" token the same as the "float64" token when used with the -type flag.  I'll add a link to the issue this fixes for better context.